### ISSUE TITLE
Transcode failure with HEVC/REXT on iGPU QSV

### ISSUE THREAD

**Issue by xsm1997** [jellyfin/jellyfin#7652]
**Media Info of the file**

https://pastebin.com/098hdjVj

**Logs**

https://pastebin.com/YhFad7dQ

**FFmpeg Logs**

https://pastebin.com/GYWgURNc

**Stats for Nerds Screenshots**

**Server System (please complete the following information):**
 - OS: Docker in Linux
 - Jellyfin Version: 10.7.7
 - Hardware settings & device: Intel QSV on Intel Pentium G5620
 - Reverse proxy: no
 - Other hardware notes: media mounted on mergerfs on hdd with bcache
 
**Client System (please complete the following information):**
 - Device: PC
 - OS: Windows 11
 - Client: Web
 - Browser (if Web client): Chrome
 - Client and Browser Version: Chrome 100.0

I cannot get HEVC/REXT transcoded with an error repeated in ffmpeg log.
```
[hevc_qsv @ 0x55e3ad2570c0] Error querying IO surface: unsupported (-3)
Error while decoding stream #0:0: Function not implemented
[hevc_qsv @ 0x55e3ad2570c0] Error initializing the MFX video decoder: invalid video parameters (-15)
Error while decoding stream #0:0: Invalid argument
```
And I'm sure it was using QSV for decoding and encoding, as:
```
Stream mapping:
  Stream #0:0 -> #0:0 (hevc (hevc_qsv) -> h264 (h264_qsv))
  Stream #0:1 -> #0:1 (flac (native) -> aac (native))
```

**Comment by xsm1997** [comment]
I can transcode the same video using VAAPI. But the transcode speed is slow compared to HEVC/10bit video (2.0x compared to 7.8x), and the log shows native not VAAPI:
```
Stream mapping:
  Stream #0:0 -> #0:0 (hevc (native) -> h264 (h264_vaapi))
  Stream #0:1 -> #0:1 (flac (native) -> aac (native))
```

BTW, when transcoding other HEVC/10bit video using VAAPI, the log also shows native, but the speed is almost the same fast as Intel QSV (QSV shows hevc_qsv). IDK it is normal behaviour or not.

**Comment by xsm1997** [comment]
The issue is resolved by upgrading to Jellyfin 10.8.0 beta 2. Thanks!
